src/engine/ruleset/decoders/sonicwall/firewall-general.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: decoder/firewall-general/0
+
+metadata:
+  module: firewall
+  title: Firewall logs decoder
+  description: Decoder for Linux firewall logs
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-12
+  references:
+    - https://www.sonicwall.com/es-mx/support/technical-documentation
+
+sources:
+  - decoder/queue-localfile/0
+
+parse:
+  logpar:
+    # Jan  3 13:45:36 192.168.5.1 id=firewall sn=000SERIAL time="2007-01-03 14:48:06" fw=89.160.20.156 pri=6 c=262144 m=98 msg="Connection Opened" n=23419 src=2.2.2.2:36701:WAN dst=89.160.20.156:50000:WAN proto=tcp/50000
+    - event.original: >-
+        <~log.timestamp/date/%b %d %T> <host.ip> id=<~log.id> sn=<~log.sn> time="<@timestamp/%F %H:%M:%6S>" fw=<~log.fw>
+        pri=<~log.pri> c=<~log.c> m=<event.code> msg="<event.action>" n=<~log.n> src=<source.ip>:<source.port>:<network.type>
+        dst=<~log.dst> proto=<network.protocol>/<~log.proto>
+    - event.original: >-
+        <~log.timestamp/date/%b %d %T> <host.ip> id=<~log.id> sn=<~log.sn> time="<@timestamp/%F %H:%M:%6S>" fw=<~log.fw>
+        pri=<~log.pri> c=<~log.c> m=<event.code> msg="<event.action>" n=<~log.n> src=<source.ip>:<source.port>:<network.type>
+        dst=<~log.dst>
+
+normalize:
+  - map:
+      - event.kind: event
+      - event.module: sonicwall
+      - event.dataset: sonicwall.firewall
+      - fileset.name: firewall
+      - input.type: log
+      - observer.product: Firewalls
+      - observer.type: Firewall
+      - observer.vendor: Sonicwall
+      - rsa.internal.messageid: $event.code
+      - rsa.time.event_time": $@timestamp
```

Comment:
why not using `rsa.time.event_time` directly on the logpar expression?
besides that Im guessing there's a typo `on _time"`